cmd/create/idp/cmd.go

```diff
@@ -434,6 +438,18 @@ func getIDPName(cmd *cobra.Command, idpName string, r *rosa.Runtime) string {
 	return strings.Trim(idpName, " \t")
 }
 
+func validateIDPName(idpName interface{}) error {
```

Comment:
@oriAdler do we adopt the naming convention that CS has, which would make it `validateIdpName` or do we stick with `validateIDPName`

Reply:
IMO should be `Idp`

Reply:
@ciaranRoche @gdbranco @oriAdler  -  So within the ROSACLI code, there are already several instances of using "IDP" , e.g doCreateIDP, getIDPName 
Also, I believe golang convention for acronyms is to use caps like in URL or ID etc

But if the consensus is to use Idp, I am happy to change it.

Reply:
Lets not block this MR over this, we should discuss it on the next office hours, and get consensus on what we adopt for ROSA